scripts/client_test.sh

```diff
@@ -65,7 +65,7 @@ elif [[ "$JOB_TYPE" == "predict" ]]; then
       --image_base=elasticdl:ci \
       --model_zoo=model_zoo \
       --model_def=mnist_functional_api.mnist_functional_api.custom_model \
-      --checkpoint_dir_for_init=model_zoo/${MNIST_CKPT_DIR}/version-100 \
+      --checkpoint_dir_for_init=/${MNIST_CKPT_DIR}/version-100 \
```

Comment:
We can also update the example command in elasticdl_local.md and elasticdl_on_prem_cluster.md.
The model zoo parameter is set as follows, we need update them too
```
 --model_zoo=/model_zoo/model_zoo \
```

Reply:
Yes!